x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -382,3 +382,10 @@ export const WARNING_TRANSFORM_STATES = new Set([
   TRANSFORM_STATES.STOPPED,
   TRANSFORM_STATES.STOPPING,
 ]);
+
+export enum INVOCATION_COUNT {
```

Comment:
Nit: could this maybe go in a more specific location with a more specific name? I don't imagine this would be shared globally.